app/templates/MembershipApplication.html.twig

```diff
@@ -52,7 +52,8 @@
 				WMDE.Components.createValidatingTextComponent( store, $( '#street' ), 'street' ),
 				WMDE.Components.createValidatingTextComponent( store, $( '#post-code' ), 'postcode' ),
 				WMDE.Components.createValidatingTextComponent( store, $( '#city' ), 'city' ),
-				WMDE.Components.createRadioComponent( store, $( '#country' ), 'country' ),
+				WMDE.Components.createSelectMenuComponent( store, $( '#country' ), 'country' ),
+				WMDE.Components.createCountrySpecificsUpdateTriggerComponent( store, $( '#country' ), 'country' ),
```

Comment:
Add multiple "components" for the same element is not how I imagined components to work.